explorer/src/components/Consensus/Extrinsic/ExtrinsicDetailsCard.tsx

```diff
@@ -64,7 +63,7 @@ export const ExtrinsicDetailsCard: FC<Props> = ({ extrinsic, isDesktop = false }
                 <StyledListItem title='Module'>{module}</StyledListItem>
                 <StyledListItem title='Call'>{call}</StyledListItem>
                 <StyledListItem title='Sender'>
-                  {isDesktop ? extrinsic.signer?.id : shortString(extrinsic.signer?.id || '')}
+                  {isDesktop ? extrinsic.signer : shortString(extrinsic.signer)}
                 </StyledListItem>
                 <StyledListItem title='Signature'>
                   {isDesktop
```

Comment:
<img width="740" alt="image" src="https://github.com/user-attachments/assets/53f1669b-6407-4bd9-8652-2c3606b9b83d">

Signature is always void does this makes? I thought that vote or set timestamp where needed signature

Reply:
Some extrinsic (like vote and timestamp) don't require being a sign extrinsic indeed

Reply:
For example, when you set up a farmer, you give it a reward address, but you don't actually give it a private key to sign these extrinsic, so these are unsigned extrinsic.